workers/unity/Packages/io.improbable.gdk.core/Systems/ComponentSendSystem.cs

```diff
@@ -65,10 +65,9 @@ protected override void OnUpdate()
 
             Profiler.EndSample();
 
-            JobHandle.CompleteAll(gatheringJobs);
-
             for (var i = 0; i < componentReplicators.Count; i++)
             {
+                gatheringJobs[i].Complete();
```

Comment:
wasn't this in the other PR

Reply:
Yeah it is, i'll fix this up once I rebase on develop